tests/frontend/test-class-wpseo-image-utils.php

```diff
@@ -55,4 +69,42 @@ public function data_get_usable_dimensions() {
 			array( 1000, 1000, true ),
 		);
 	}
+
+	/**
+	 * Tests if the absolute path is working as expected
+	 *
+	 * @param string $input    Data to use in execution.
+	 * @param string $expected Expected result.
+	 * @param string $reason   Description of the tested data.
+	 *
+	 * @dataProvider data_get_absolute_path
+	 */
+	public function test_absolute_path( $input, $expected, $reason = '' ) {
+		$result = WPSEO_Image_Utils::get_absolute_path( $input );
+
+		$this->assertEquals( $expected, $result, $reason );
+	}
+
+	/**
+	 * Provides data for the absolute path test.
+	 *
+	 * @return array Data.
+	 */
+	public function data_get_absolute_path() {
+		$uploads = wp_get_upload_dir();
+
+		return array(
+			array(
```

Comment:
I thought each test group should be in its own array. Now you have 9 values in one array for three tests. Is that okay?